manifest.json

```diff
@@ -1,5 +1,10 @@
 {
 	"manifest_version": 2,
+	"browser_specific_settings": {
+		"gecko": {
```

Comment:
What does this do?

Reply:
This is so that I can manually load the extension in Firefox. They have two options. The first option is to load the extension temporarily. The other option is to install an extension from a package instead of from the firefox store and this option requires an extension ID. There are also other potential uses for the ID in the future: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID